content/azure-pipeline-blueprints-oss-sync-20240322.md

```diff
@@ -0,0 +1,28 @@
+Title: Azure DevOps Pipeline Blueprints - Open Source Software Sync 20240322
+Date: 2024-04-01
+Category: Posts
+Tags: azure-devops, pipelines, engineering
+Slug: azure-pipeline-blueprints-oss-sync-20240322
+Author: Willy-Peter Schaub
+Summary: Second update to complete the **101** simplification.
+
+This open-source update is minor, but aligns our NuGet-Package blueprint with the recent [Azure DevOps Pipeline Blueprints - Open Source Software Sync 20240318](https://wsbctechnicalblog.github.io/azure-pipeline-yaml-refactor-parameter-objects.html) simplifications. Now all blueprints are consistent!
+
+---
+
+>
+> [Click here to view the associated Pull Request](https://github.com/WorkSafeBC-Common-Engineering/AzureDevOps.Automation.Pipeline.Templates.v2/pull/27)
+>
+
+Changes:
+
+- **blueprints/nuget-package/azure-pipeline-nuget-package-cd-stage.yml** - eplace the configuration variables with the ```config``` object, which encapsulates an array of data structures.. 
```

Comment:
missing the 'r' in 'eplace' (first word)

Reply:
and extra period at end.

Reply:
Thank you @MerlinDarkstar. Fixed.